### ISSUE TITLE
unsupervised video segmentation

### ISSUE THREAD

**Issue by lbbcalm** [Hydragon516/GSANet#1]
Hi, I have a question, can you answer it? Why is it called unsupervised video segmentation when the target frame is supervised by the truth value?

**Comment by zhending111** [comment]
Whether he is unsupervised or not depends on whether the input is given to the first frame during the test

**Comment by dipikaboro2** [comment]
Hello. Congratulations on the great work.
I am new to unsupervised learning in Computer Vision. My question is - 
1. In Unsupervised VOS is it OK to compute loss using ground truth during training or pre-training?
2. In your previous answer you mentioned that supervised or unsupervised depends on the whether mask is given for the first frame for test data. However, in the codebase, I found that for all the RGB images, the corresponding masks (GT) are also provided during testing. [ https://github.com/Hydragon516/GSANet/blob/1a70dada4bb2599db6b2f60eadba496378280c9c/train.py#L111 ]
